projects/arcan/default.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+  # NOTE:
+  # - Check `projects/models.nix` for a more detailed project structure
+  # - Each program/service must have at least one example
+  # - Set attributes to `null` to indicate that they're needed, but not available
+  metadata = {
+    summary = "Explorative p2p protocol for fast and secure remote desktops";
+    subgrants = [
+      "arcan-A12"
+      "arcan-A12-directory"
+      "arcan-A12-tools"
```

The names should **exactly** match the ones in the NLnet page

```suggestion
      "Arcan-A12"
      "Arcan-A12-directory"
      "Arcan-A12-tools"
```